internal/testutil/context.go

```diff
@@ -133,6 +133,13 @@ func JWTConfig() (*jwt.Config, error) {
 	return jwtConfigFromFile(os.Getenv(envPrivateKey), nil)
 }
 
+// JWTConfigExplicit reads an explicit credentials file
+// that is passed in as a parameter. It ignores scopes.
+// If the environment variable is empty, it returns (nil, nil).
+func JWTConfigExplicit(credFile string) (*jwt.Config, error) {
```

Comment:
nit: I'd just export the original `jwtConfigFromFile` or create a function in Storage instead.